integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/schema/CarbonAlterTableRenameCommand.scala

```diff
@@ -61,6 +61,8 @@ private[sql] case class CarbonAlterTableRenameCommand(
     }
     val oldTableName = oldTableIdentifier.table.toLowerCase
     val newTableName = newTableIdentifier.table.toLowerCase
+    val oldIdentifier = TableIdentifier(oldTableName, Some(oldDatabaseName))
```

Comment:
what the purpose of creating oldIdentifier and newIdentifier again?? when already both identifiers are present
https://github.com/apache/carbondata/pull/3098/files#diff-9fb2c5a8b63af8d61563a154cdfc8a3cR46

Reply:
Done refactoring for this.